lib/main.js

```diff
@@ -47,6 +48,19 @@ const fixPathString = (pathString, fileDir, projectDir) => {
   return pRstring;
 };
 
+const resolvePath = (fullPath) => {
+  const paths = fullPath.split(';');
+  if (paths.length === 1) {
+    return paths[0];
+  }
+  for (let i = 0; i < paths.length; i += 1) {
+    if (fs().existsSync(paths[i])) {
+      return paths[i];
```

Comment:
Since we don't want to be calling this synchronous `stat` call on _every_ `lint()` call the results of this should be cached for each path given.